### PR TITLE
Cherrypick-RN-enterprise-4.9: Added 1.7.0 RN and resolved merge confl…

### PR DESCRIPTION
**Pull request by shipsing**
**Aligned team**: Dev Tools

**Purpose**: Cherrypick 1.7.0 RN to enterprise 4.9 to resolve future conflicts in context of this issue: https://issues.redhat.com/browse/RHDEVDOCS-4940, which needs to be cherrypicked to enterprise-4.9

Cherrypicking of GitOps 1.7.0 RN (https://github.com/openshift/openshift-docs/pull/53541) failed for enterprise 4.9 due to merge conflicts and We could not create a branch against main because the main branch already had this file `gitops-release-notes-1-7-0.adoc`. So, created a new branch against the respective branch, enterprise-4.9, to cherrypick 1.7.0 RN content and resolve merge conflicts.

**OCP version this PR applies to**: enterprise 4.9

Preview link: [Download GitOps RN for preview](https://drive.google.com/file/d/1d0Y5xUPtXM3k0ryblhAXBtBA3yxh8Kbr/view?usp=sharing)

Peer review: @Srivaralakshmi

### CODE REVIEW

**Comment by shipsing** [comment]
Added the 1.7.0 RN and resolved merge conflicts in the following two files:

- **go-compatibility-and-support-matrix.adoc**: OCP versions 4.8 and 4.9 use this old table: https://docs.openshift.com/container-platform/4.8/cicd/gitops/gitops-release-notes.html#compatibility-and-support-matrix, wheras OCP 4.10 and later uses this new table: https://docs.openshift.com/container-platform/4.10/cicd/gitops/gitops-release-notes.html#compatibility-and-support-matrix. So, i replaced the old table with  new one in OCP versions 4.8 and 4.9.

- **gitops-release-notes.adoc**: The modules included varies for OCP versions 4.8 and 4.9 if compared to 4.10 and later. The RN modules for 1.4.11 and 1.4.6 were missing from OCP versions 4.8 and 4.9. So, i added these 2 modules: gitops-release-notes-1-4-11.adoc and gitops-release-notes-1-4-6.adoc. 

In OCP version 4.8, we have an additional RN module for GitOps 1.2.2 (https://docs.openshift.com/container-platform/4.8/cicd/gitops/gitops-release-notes.html#gitops-release-notes-1-2-2_gitops-release-notes) and the same module was not available in 4.9 and later. @Preeticp either we need to delete this RN module from OCP version 4.8 or add this RN module in OCP version 4.9 and later. waiting for your response. This needs to be decided. Otherwise, we would face a merge conflict for OCP version 4.8 because that module was not included in 4.9 and later.

When the above query is resolved, we are good to merge this PR. @Preeticp please have a look at this PR.

File: modules/go-compatibility-and-support-matrix.adoc
Comment (Srivaralakshmi) [inline modules/go-compatibility-and-support-matrix.adoc:27]: **Question:** Any particular reason why this Note related to OCP 4.6, below the matrix table, is removed?

**Observation:** I do not see this Note being covered elsewhere. 

**Suggestion:** Consult with SMEs whether we need to include or remove this Note in all the OCP versions of GitOps RN.

File: modules/go-compatibility-and-support-matrix.adoc
Comment (Srivaralakshmi) [inline modules/go-compatibility-and-support-matrix.adoc:23]: **Question:** Any particular reason why these rows are removed?

**Suggestion:** Retain these in all the OCP versions of GitOps RN

File: modules/go-compatibility-and-support-matrix.adoc
Comment (Srivaralakshmi) [inline modules/go-compatibility-and-support-matrix.adoc:30]: @shipsing **Observation:**  [GITOPS-1756](https://issues.redhat.com/browse/GITOPS-1756) is already documented in 4.8 OCP under Fixed Issues. 

**Suggestion:**

- So no need of repeating it in RN 1.4.2 and 1.3.3. 

- Also, add the RN module 1.2.2 in OCP version 4.9 and later.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@shipsing Nice work and detailed research!! Good job!

I have given my review comments. PTAL. I am holding back my observations and comments on the support matrix table versions and queries (https://docs.google.com/document/d/1c4Q_osMeV7KshsY3yc9OzWLyhgGCx0SC4BPQl21SA4g/edit) as per our 1:1 discussion. Rest of the content LGTM.

Please discuss this with Preeti and let me know if we can take it as a separate issue and resolve it with the help of SMEs. We definitely need to discuss this with the concerned SMEs and get clarity.

File: modules/go-compatibility-and-support-matrix.adoc
Comment (shipsing) [reply modules/go-compatibility-and-support-matrix.adoc:18]: new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958

File: modules/go-compatibility-and-support-matrix.adoc
Comment (shipsing) [reply modules/go-compatibility-and-support-matrix.adoc:19]: new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958

File: modules/go-compatibility-and-support-matrix.adoc
Comment (shipsing) [reply modules/go-compatibility-and-support-matrix.adoc:27]: new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958

File: modules/go-compatibility-and-support-matrix.adoc
Comment (shipsing) [reply modules/go-compatibility-and-support-matrix.adoc:23]: new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958

**Comment by shipsing** [comment]
@Srivaralakshmi, Thank you for reviewing this PR. I created a separate Jira issue to track and incorporate all of these comments associated with this file [go-compatibility-and-support-matrix.adoc]. Now, if you are good with the changes in this PR, can we route this PR for merge? This PR needs to be merged before we merge the PR related to CVE fixes: https://github.com/openshift/openshift-docs/pull/55464.

File: modules/go-compatibility-and-support-matrix.adoc
Comment (shipsing) [reply modules/go-compatibility-and-support-matrix.adoc:30]: new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958

**Comment by Srivaralakshmi** [comment]
> @Srivaralakshmi, Thank you for reviewing this PR. I created a separate Jira issue to track and incorporate all of these comments associated with this file [go-compatibility-and-support-matrix.adoc]. Now, if you are good with the changes in this PR, can we route this PR for merge? This PR needs to be merged before we merge the PR related to CVE fixes: #55464.

@shipsing Yes please. LGTM from my side. Thanks for the good work!

**Review by Srivaralakshmi — APPROVED** [review]

File: modules/go-compatibility-and-support-matrix.adoc
Comment (Srivaralakshmi) [inline modules/go-compatibility-and-support-matrix.adoc:22]: @shipsing Just try and retain these two rows for 1.2.0 and 1.1.0

File: modules/go-compatibility-and-support-matrix.adoc
Comment (shipsing) [reply modules/go-compatibility-and-support-matrix.adoc:30]: Resolved in this PR: https://github.com/openshift/openshift-docs/pull/55464#discussion_r1100082257

**Pull request merged**
